features/machine/OCP24363.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Machine taints
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario Outline: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I clone a machineset named "machineset-24363"
```

Comment:
what is this machine-set? Do we have such step?

Reply:
Wanted to execute test case on new machines , hence created a new machineset , so that , it will be removed once the scenario is completed , making sure the cluster goes back to initial state , after the execution

Reply:
For this step, I think you don't have to embed it in a wait for. It has wait logic in it already.

Reply:
removed wait